### ISSUE TITLE
Java "import static"

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#935]
(Original issue 1228 created by benromberg on 2016-04-13T16:37:32.119879+00:00)

In Java, when using a so-called [static import](https://en.wikipedia.org/wiki/Static_import), the `static` keyword should be highlighted as well. Example:
```java
import static java.util.Arrays.asList;
```

Not even Bitbucket highlights this correctly - are they using Pygments? It seems that [GitHub is able to do this](https://github.com/fakemongo/fongo/blob/cb365f47e9dda91d5cfb19ca977f9379f9e23cf6/src/test/java/com/github/fakemongo/integration/SpringMongoOperationTest.java) though.

Not sure how this would be implemented, maybe by changing [the line](https://bitbucket.org/birkenfeld/pygments-main/src/859b2bce647cd212e39cd63f83f93f4083b232b2/pygments/lexers/jvm.py?at=default&fileviewer=file-view-default#jvm.py-63)
```python
(r'(import)(\s+)', bygroups(Keyword.Namespace, Text), 'import'),
```
to
```python
(r'(import|import static)(\s+)', bygroups(Keyword.Namespace, Text), 'import'),
```
I'm not proficient with python though...

**Comment by Anteru** [comment]
(Original issue 1228 last updated on 2016-04-14T07:39:35.683271+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2016-04-14T06:19:54.640590+00:00)

Java lexer: add "import static" (fixes #1228)

→ <<cset c6b00b825afa>>

**Comment by Anteru** [comment]
(Original comment by benromberg on 2016-04-14T07:39:35.675563+00:00)

Splendid, thanks for the swift resolution!
